main.go

```diff
@@ -130,20 +130,36 @@ func processGeofeed(geofeedFilename, mmdbFilename string) (counts, []string, err
 	csvReader := csv.NewReader(geofeedFH)
 	csvReader.ReuseRecord = true
 	csvReader.Comment = '#'
-	csvReader.FieldsPerRecord = 5
+	csvReader.FieldsPerRecord = -1
 	csvReader.TrimLeadingSpace = true
 
+	const expectedFieldsPerRecord = 5
+
+	// We will assume one record per line. Multi-line records are valid
+	// so this assumption isn't guaranteed to hold up; but with only five
+	// fields per record, asking the geofeed producer to restrict themselves
+	// to one record per line seems reasonable.
+	rowCount := 0
+
 	for {
 		row, err := csvReader.Read()
 		if err == io.EOF {
 			break
 		}
+		rowCount++
 		if err != nil {
 			return c, diffLines, err
 		}
+		if len(row) < expectedFieldsPerRecord {
+			return c, nil, fmt.Errorf(
+				"saw fewer than the expected %d fields at line %d",
+				expectedFieldsPerRecord,
+				rowCount,
+			)
+		}
 
 		c.total++
-		currentCorrectionCount, diffLine, err := verifyCorrection(row, db)
+		currentCorrectionCount, diffLine, err := verifyCorrection(row[0:expectedFieldsPerRecord-1], db)
```

Comment:
I think we are dropping the postal code here.

https://play.golang.org/p/Fj6HbWLQ2N7

Reply:
Yep, I forgot that the end of the range is exclusive, so that was an error. I was previously not looking for postal code differnces since it could create a lot noise, but in 96e4b7597f78ceac73c1bf71a6c991dff036d5e5 we now only compare the postal codes if the correction includes one.  For all things we are comparing, we now only print out the differences to help further reduce noise in the output.